core/table.ts

```diff
@@ -217,6 +233,7 @@ export interface ITableContext {
   ) => string;
   describe: (key: string, description?: string) => string;
   apply: <T>(value: TContextable<T>) => T;
+  tags: (name: string[]) => string;
```

Comment:
this should probably have the same type signature as the other `Context` types, i.e. `(name: string | string[]) => string`

Reply:
Yes. BTW, this is off-topic but on line 228 you have: 

` dependencies: (name: string) => string;`

So perhaps that one needs to be changed to this:

` dependencies: (name: string | string[]) => string;`

Reply:
Ah, yes good catch, we should clean that up!